projects/demo-maps/src/app/route-components/route-example-custom-layers/route-map4.component.ts

```diff
@@ -49,7 +52,6 @@ export class RouteMap4Component implements OnInit, AfterViewInit {
 
 
   ngOnInit(): void {
-    this.mapSvc.setProjection('EPSG:4326');
```

Comment:
nice that you removed this, i noticed it is causing a problem with the Vector Tile layer 👍

Reply:
Oh! Sorry, I had not noticed that. In that case, yeah, good riddance :)